qiita_db/sql_connection.py

```diff
@@ -402,9 +430,16 @@ def execute_fetchall(self, sql, sql_args=None):
         """
         with self._sql_executor(sql, sql_args) as pgcursor:
             result = pgcursor.fetchall()
-        return result
+            if return_types:
```

Comment:
First option would be painful and require changing every execute_fetch* call. Second option is doable.